README_ZH.md

```diff
@@ -17,9 +17,16 @@ dev  默认分支
 
 ## 1.预览并生成静态文件
 
-本网站是使用 Node.js 编译的，使用的是 [Docusaurus](https://docusaurus.io/) 框架组件：
-
-1. 下载并安装 Node.js（version>12.5.0）
+本网站是使用 Node.js 编译的，使用的是 [Docusaurus(2.4.3)](https://docusaurus.io/) 框架组件：
+
+1. 下载并安装 Node.js,支持的版本列表如下:
+   
+   |  OS     | NodeJs       | 
+   |  ----   | ----         |
+   | Windows | 16.14 ~ 18   |
+   | OSX     | 16.14 ~ 18   |
+   | Linux   | 16.14 ~ 18   |
+   
```

Comment:
If the required node versions are all the same, I don’t think there is any need to write this list. The explanation is version>16.14&&version<16.18